src/appendices/git/stash.rst

```diff
@@ -0,0 +1,113 @@
+.. _git-stash:
+
+Git Stash
```

Comment:
This section is well laid out and reads very smoothly.  I have no updates.  Woo hoo!